### ISSUE TITLE
Don't display synopsis asterisk in Topics

### ISSUE THREAD

**Issue by olsonjaredm** [FodWikis/FodWikis#6]
If bulleted list is used in minutes synopsis, under Topics summary view the first bullet is displayed as an asterisk and should be ignored instead.
![image](https://cloud.githubusercontent.com/assets/13458815/8988810/0e3ad3ae-36ae-11e5-8794-0fbcecdb9923.png)

**Comment by jamesmontalvo3** [comment]
You're on an old version of meeting minutes that handles synopsis differently. Add another divergent configuration to the list :(

**Comment by olsonjaredm** [comment]
Yes, I figured there are several things like this that will be fixed when ROBO upgrades minutes to match EVA since we're on a pretty old version.
I guess what I should do is watch the templates on EVA wiki and copy the contents over when it changes.

**Comment by olsonjaredm** [comment]
I checked after the minutes upgrade and issue still exists.
The examples I saw are on:
https://mod2.jsc.nasa.gov/wiki/ROBO/index.php?title=FRAT_Leadership_-_2015/07/30
https://mod2.jsc.nasa.gov/wiki/ROBO/index.php?title=MIP_-_2015/06/23
